src/Tools/dotnet-monitor/Egress/EgressArtifactSettings.cs

```diff
@@ -5,27 +5,17 @@ namespace Microsoft.Diagnostics.Tools.Monitor.Egress
 {
     internal sealed class EgressArtifactSettings
     {
-        /// <summary>
-        /// The content encoding of the blob to be created.
-        /// </summary>
-        public string ContentEncoding { get; set; }
-
         /// <summary>
         /// The content type of the blob to be created.
         /// </summary>
         public string ContentType { get; set; }
+
         /// <summary>
         /// The metadata of the blob to be created.
         /// </summary>
         public Dictionary<string, string> Metadata { get; }
             = new Dictionary<string, string>(StringComparer.Ordinal);
 
-        /// <summary>
-        /// Custom metadata of the blob to be created.
-        /// </summary>
-        public Dictionary<string, string> CustomMetadata { get; }
```

Comment:
The concept of `CustomMetadata` only pertains to the `AzureBlobStorage` extension, so this should not be part of `dotnet-monitor`'s extension contract.